lib/index.js

```diff
@@ -170,29 +197,36 @@ internals.stop = function (server, next) {
         return next();
     }
 
-    return Knex.destroy(next);
+    return internals.state(server).knex.destroy(next);
 };
 
 internals.schwifty = function (config) {
 
     let isModels = false;
 
-    // Array of models, coerce to config
     if (Array.isArray(config)) {
+
+        // Array of models, coerce to config
+        config = Joi.validate(config, Schema.schwifty);
+        isModels = true;
+    }
+    else if (typeof config === 'function' && config.tableName) {
```

Comment:
Maybe something like `config.prototype instanceof Objection.Model` as opposed to duck-typing?